Lib/asyncio/streams.py

```diff
@@ -1351,6 +1351,11 @@ def is_server_side(self):
 
     @property
     def transport(self):
+        warnings.warn("Stream.transport attribute is deprecated "
```

Comment:
asyncio itself never use this property, there is no need for readonly internal API version